src/plugins/LinkManager/components/LinkModal/LinkModal.tsx

```diff
@@ -0,0 +1,56 @@
+import { InputEditModal } from "@kaizen/draft-modal"
+import { TextField } from "@kaizen/draft-form"
+import React, { useRef, useState } from "react"
+
+interface LinkModalProps {
+  onSubmit: (href: string) => void
+  onDismiss: () => void
+  onAfterLeave: () => void
+  isOpen: boolean
+  defaultHref?: string
+}
+
+export const LinkModal: React.VFC<LinkModalProps> = props => {
+  const { onSubmit, onDismiss, onAfterLeave, isOpen, defaultHref } = props
```

Comment:
Minor - Could line 14 be de-structured in line 13 replacing props with `({ onSubmit, onDismiss, onAfterLeave, isOpen, defaultHref })`

Reply:
I personally prefer it this way. It's not always useful, but it does allow to work with the props object as a whole if you ever want to. There might be the odd case where I don't want to destructure a prop and instead reference it from the props object.

For example, I might want to reference a prop in a component, but still have it be part of the `...restProps` spread:
```
export const LinkModal: React.VFC<LinkModalProps> = props => {
  const { onSubmit, onDismiss, onAfterLeave, isOpen, defaultHref, ...restProps } = props
  const someConditional = props["aria-labelledby"] != null
}
```

I wouldn't be able to do that with the props destructured in the function declaration. Or even if I just want to console.log all the props for some reason.